src/gleam_stdlib.erl

```diff
@@ -27,6 +27,12 @@
            ((X) >= $a) andalso ((X) =< $f) -> (X) - $a + 10
         end).
 
+-define(is_lowercase_char(X), (X > 96 andalso X < 123)).
+
+-define(is_underscore_char(X), (X == 95)).
+
```

Comment:
```suggestion
```

Reply:
I will remove these once they are not required anymore.